source/isaaclab/isaaclab/envs/manager_based_rl_env.py

```diff
@@ -107,6 +109,7 @@ def max_episode_length(self) -> int:
     """
 
     def load_managers(self):
+
         # note: this order is important since observation manager needs to know the command and action managers
```

Comment:
```suggestion
    def load_managers(self):
        # note: this order is important since observation manager needs to know the command and action managers
```